.github/workflows/release-please-v1.yml

```diff
@@ -1,8 +1,14 @@
 ---
-name: Manage releases using release-please
+name: Manage releases
 
 on:
   workflow_call:
+    inputs:
+      use_release_pr:
```

Comment:
I would rename to `create_pull_request` instead as we already are in the context of a release so this is kind of redundant